apps/daffio/src/app/docs/shared/components/doc-viewer/doc-viewer.component.ts

```diff
@@ -12,11 +16,18 @@ import { DaffioDoc } from '../../models/doc';
   templateUrl: './doc-viewer.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class DaffioDocViewerComponent {
+export class DaffioDocViewerComponent implements OnChanges {
+
+  constructor(private sanitizer: DomSanitizer) {}
 
   /**
    * The doc to render
    */
   @Input() doc: DaffioDoc;
 
+	sanitizedContent: SafeHtml;
+
+	ngOnChanges() {
+	  this.sanitizedContent = this.sanitizer.bypassSecurityTrustHtml(this.doc.contents);
```

Comment:
Can you add a comment outlining why this is necessary?

Reply:
added